src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+import static java.lang.Math.PI;
+
+public class Circle extends Figure {
+
```

Comment:
this empty line contradict other class
we should agree to have empty line after class definition everywhere or not to have it everywhere

Reply:
I think I understood correctly